libs/sdk-ui-dashboard/src/plugins/customizer.ts

```diff
@@ -135,6 +138,9 @@ export interface IDashboardKpiCustomizer {
      *
      *         function MyCustomDecorator() {
      *              const Decorated = next(kpi, widget);
+     *              if (!Decorated) {
+     *                  return null;
+     *              }
```

Comment:
hmm, ins't there a problem with typing of next(); decorator should always work on top of something, the underlying infra should ensure that.

Reply:
Fixed by improving the typings in the area